stdlib/public/core/Sequence.swift

```diff
@@ -384,16 +384,25 @@ public protocol Sequence {
     initializing ptr: UnsafeMutableBufferPointer<Element>
   ) -> (Iterator,UnsafeMutableBufferPointer<Element>.Index)
   
-  /// Call `body(p)`, where `p` is a pointer to the collection's
-  /// contiguous storage.  If no such storage exists, it is
-  /// first created.  If the collection does not support an internal
-  /// representation in a form of contiguous storage, `body` is not
-  /// called and `nil` is returned.
+  /// Executes a closure on the sequence’s contiguous storage.
+  ///
+  /// This method calls `body(p)`, where `p` is a pointer to the collection’s
+  /// contiguous storage. If no such storage exists, the collection creates it.
+  /// If the collection doesn’t support an internal representation in a form of
+  /// contiguous storage, the method doesn’t call `body` and returns `nil`.
   ///
   /// A `Collection` that provides its own implementation of this method
-  /// must also guarantee that an equivalent buffer of its `SubSequence` 
-  /// can be generated by advancing the pointer by the distance to the
-  /// slice's `startIndex`.
+  /// must also guarantee that advancing the pointer by the distance to the
```

Comment:
```suggestion
  /// must also guarantee that advancing the pointer by the distance to a
```

It's unclear what would be referred to here by "_the_ slice." Overall, I feel the original wording was more easily comprehended here.